bedrock/exp/templates/exp/firefox/accounts.html

```diff
@@ -0,0 +1,200 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros-protocol.html" import feature_card with context %}
+{% from "macros.html" import fxa_email_form with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% set _entrypoint = 'mozilla.org-firefox-accounts' %}
+{% set _utm_source = 'mozilla.org-firefox-accounts' %}
+{% set _utm_campaign = 'trailhead' %}
+
+{%- block page_title -%}
+  {{ ftl('firefox-accounts-there-is-a-way-to', fallback='firefox-accounts-get-a-firefox-account') }}
```

Comment:
note for reviewer: I know that our exp templates should not be a part of our localization system-- is there a way to retrieve the strings from these fluent ID's? (have not had a chance to interact with the new fluent system very much)

Reply:
@amychurchwell the strings for the associated page can be found here: https://github.com/mozilla/bedrock/blob/master/l10n/en/firefox/accounts.ftl

It's up to you if you want to go as far as replacing them really. The docs are just meant as a guide to reinforce that experimental pages can be standalone and lower maintenance.

Reply:
Ah, okay. I'll leave this as-is then. Thank you.